languages/languages-spec.md

```diff
@@ -49,38 +49,38 @@ There are 4 types of status:
 - PR reviewing: a translator finished the translation, and the work is being reviewed.
 - PR merged: the translation is successfully merged into the `spec` repository.
 
-Initially, the last edit time will be the time when the translation is merged. The time format 
+Initially, the last edit time will be the time when the translation is merged. The time format
 MUST be the UTC format (YYYY-MM-DDTHH:mm:ss.sssZ).
 
-Reviewers SHOULD ensure translations are up-to-date with the English version as it changes, meanwhile 
-update the last edit time as well. 
+Reviewers SHOULD ensure translations are up-to-date with the English version as it changes, meanwhile
+update the last edit time as well.
 
 The following rules govern adding and removing reviewers:
 
 - Reviewers can be proposed via a PR to edit the `/docs/languages.md` file.
 
 - Reviewers can be removed via a PR to edit the `/docs/languages.md` file.
 
-## How to add a new language 
+## How to add a new language
 
 This section is mainly for anyone who needs to build a new language structure.
 
 Firstly, check the [language list](../docs/languages.md) to see if the language
-you'd like to translate into already exists. If it doesn't exist, 
+you'd like to translate into already exists. If it doesn't exist,
 and there is not an issue indicating someone is currently working on it,
 then you can start to add a new one.
 
 ### Step 1. Open an issue to avoid redundant works
 
-Before starting to do any actual works, you should open an issue to explicitly declare 
+Before starting to do any actual works, you should open an issue to explicitly declare
 what you want to do.
 
 ### Step 2. Build the structure locally
 
-Currently, each spec directory(/cloudevents, /discovery, or /cesql, etc) 
+Currently, each spec directory(/cloudevents, /registry, or /cesql, etc)
```

Comment:
Add a space before the bracket, while we're here?

Reply:
done